Dockerfile.deploy.rhel

```diff
@@ -11,7 +11,7 @@ ADD nginx.conf /etc/nginx/nginx.conf
 COPY scripts/entrypoint.sh /usr/bin/entrypoint.sh
 RUN chmod +x /usr/bin/entrypoint.sh
 
-RUN rm /usr/share/nginx/html/*
+RUN rm -f /usr/share/nginx/html/*
```

Comment:
Shouldn't `-f` be added to `Dockerfie.deploy` as well?

Reply:
I checked and it's not strictly necessary, but let me refresh this PR and submit it in any case, to make them both be as similar as possible. The reason why it's not strictly necessary is because there are subtle differences between the the centos based container referenced in the FROM and the RHEL one.

Reply:
submitted.